client/app/bundles/shared/components/Textarea.scss

```diff
@@ -0,0 +1,54 @@
+@import "base.scss";
+
+$focus-background-color-darken: rgba(0, 0, 0, 0.4);
+$focus-background-color-lighten: lighten($container-color, 3%);
+$light-font-weight: 300;
+
+.label {
+  color: $container-color;
+  font-family: $font-family;
+  font-size: $small-font;
+  padding: 12px 0;
+  letter-spacing: 0.0944444em;
+  text-indent: 4px;
```

Comment:
Let's make this a padding, since when the label breaks onto multiple lines on smaller screens it looks a little funky!

Reply:
Thanks for noticing that! I also ended up adjusting the padding and line height in the input portion, just a heads up! Hopefully made it easier to read.